### ISSUE TITLE
getting ip6tables error, both Ubuntu and CentOS hosts

### ISSUE THREAD

**Issue by carlosrmendes** [squat/kilo#55]
Using kilo as cni in kubeadm (`kilo/manifests/kilo-kubeadm.yaml` file), I'm getting the following error in all nodes (ubuntu18.04 and centos7):
```
{"caller":"mesh.go:618","component":"kilo","error":"failed to add rule: failed to add iptables chain: running [/sbin/ip6tables -t nat -N KILO-NAT --wait]: exit status 3: modprobe: can't change directory to '/lib/modules': No such file or directory\nip6tables v1.8.4 (legacy): can't initialize ip6tables table `nat': Table does not exist (do you need to insmod?)\nPerhaps ip6tables or your kernel needs to be upgraded.\n","level":"error","ts":"2020-05-07T13:59:26.463594297Z"}
```

But after than I execute the command `/sbin/ip6tables -t nat -N KILO-NAT --wait` directly on the node kilo starts working, configuring correctly the pod network and wireguard conf.

**Comment by squat** [comment]
Ah yes thanks for filing this! Indeed i had seen this before and when debugging it I had fixed it for myself. The fix is in the debug message: we need to mount /lib/modules from the host into the kilo container :). This will simply be a question of modifying the manifests. Would you like to make the PR? otherwise I can do it

**Comment by ibalajiarun** [comment]
@squat Is it possible to version kilo images to prevent such breakages? I create temporary clusters frequently and I have an ansible playbook to bring up clusters with kilo. Even though I have a copy of the manifest locally, I have noticed that images updates have frequently lead to breakages. I understand the product is rapidly evolving, but I hope versioning docker images should solve such issues.

**Comment by squat** [comment]
Hi @ibalajiarun, while Kilo does not yet have a tagged release, every commit hash _is also_ a valid docker image tag. For example if you want to pin to the state of master today, you could use the image: squat/kilo:968d13148fe6b9b1e1876eee4afcd16c11ad722b

You can find all image tags here: https://hub.docker.com/r/squat/kilo/tags

**Comment by squat** [comment]
@carlosrmendes this should be fixed now by b188abf0b6be3b62f45951a6cb26cc6e107d3dd3 :)
